src/fs/ufs/UFSSwapDir.cc

```diff
@@ -1078,7 +1079,7 @@ Fs::Ufs::UFSSwapDir::HandleCleanEvent()
          * value.  j equals the total number of UFS level 2
          * swap directories
          */
-        std::mt19937 mt(static_cast<uint32_t>(getCurrentTime() & 0xFFFFFFFF));
+        static std::mt19937 mt(Seed32());
```

Comment:
This change avoids creation of large std::mt19937 objects every time we run this UFS cleaning code, but I think this "optimization" is not significant enough to warrant a PR description mention.